tests/LimitOrderTest.php

```diff
@@ -22,6 +22,7 @@ public function testBasic()
         $i = (new Model($this->db, ['table' => 'invoice']))->addFields(['total_net', 'total_vat']);
         $i->addExpression('total_gross', '[total_net]+[total_vat]');
         $i->getField($i->id_field)->system = false;
```

Comment:
these lines here and below probably are no more needed if you set id_field=null

Reply:
They are needed, only test does not pass - this does test `onlyFields`, but ID must be undefined if not flagged as a system. (normally, you keep it flagged as system field and this is not needed)